pom.xml

```
@@ -141,7 +140,7 @@
 						<image>openjdk:14</image>
 					</from>
 					<to>
-						<image>${image.prefix}${project.artifactId}</image>
+						<image>${image.prefix}/${project.artifactId}</image>
```

Can revert all the way to the following so that an unset `image.prefix` leaves this syntactically correct
```suggestion
						<image>${image.prefix}${project.artifactId}</image>
```